### ISSUE TITLE
Line has no efect

### ISSUE THREAD

**Issue by chrissivo** [rnd-ash/ultimate-nag52-fw#51]
This line has no effect, since the hold_time is later set to 100. One of these lines should be deleted.
https://github.com/rnd-ash/ultimate-nag52-fw/blob/064e292a6e8d6547d70306be2c74b05780ba1dbb/src/pressure_manager.cpp#L240

**Comment by rnd-ash** [comment]
Yes agreed. This is already resolved in the shift_pressure_dynamic_offset branch. (As the entire pressure calc logic changed). Once stabilized a bit more, it will be merged into dev for wider testing, then main
